mne/utils/_bunch.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+"""Bunch-related classes."""
```

Comment:
Do we actually use the plain `Bunch` and `BunchConst` classes anywhere? Why do we need the tree of subclasses?